frontend/libs/studio-components/src/components/StudioAvatar/StudioAvatar.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { type HTMLAttributes, type ReactElement } from 'react';
+import classes from './StudioAvatar.module.css';
+import { PersonCircleIcon } from '@studio/icons';
+
+export type StudioAvatarProps = {
+  avatarElement?: HTMLAttributes<HTMLImageElement>;
+};
+
+export const StudioAvatar = ({ avatarElement }: StudioAvatarProps): ReactElement => {
+  if (!avatarElement) {
+    return <PersonCircleIcon className={classes.avatar} />;
+  }
+  return <img {...avatarElement} className={classes.avatar} />;
+};
```

Comment:
We should add the classname from the avatarElement as well. 

```suggestion
export const StudioAvatar = ({ avatarElement }: StudioAvatarProps): ReactElement => {
  if (!avatarElement) {
    return <PersonCircleIcon className={classes.avatar} />;
  }
  return <img {...avatarElement} className={cn(classes.avatar, avatarElement.className)} />;
};
```